public/content/lessons/2016/cross-products-extended/index.mdx

```diff
@@ -1,10 +1,189 @@
 ---
 title: Cross products in the light of linear transformations
-description: The formula for the cross product can feel like a mystery, or some kind of crazy coincidence.  But it isn't.  There is a fundamental connection between the cross product and determinants.
+description: The formula for the cross product can feel like a mystery, or some kind of crazy coincidence. But it isn't. There is a fundamental connection between the cross product and determinants.
 date: 2016-09-01
 chapter: 11
 video: BaM7OCEm3G0
 source: _2016/eola/chapter8.py
 credits:
 - Lesson by Grant Sanderson
 ---
+
+> "From [Grothendieck], I have also learned not to take glory in the difficulty of a proof: difficulty means we have not understood. The idea is to be able to paint a landscape in which the proof is obvious."
+>
+> \- Pierre Deligne
+
+Where we left off, I was talking about how to compute a three-dimensional cross product of two vectors, v cross w. It's this funny thing where you write a matrix whose second column has the coordinates of v, whose third columns has the coordinates of w, but the entries of the first column, weirdly, are the basis vectors i hat, j hat and k hat, where you just pretend like those guys are numbers for the sake of computations. If you just chug along with the computation, ignoring this weirdness, you get some constant times i hat, plus some constant times j hat, plus some constant times k hat, which defines a new 3d vector.
```

Comment:
Replace some of the prose description of vectors with tex, 
- "v cross w" -> "$\mathbf{v} \times \mathbf{w}$"
- "i hat" -> "$\hat{imath}$"
- etc.